src/test/end-to-test/test_suites/basic_options_test.ts

```diff
@@ -0,0 +1,237 @@
+import type ParsedGameOptionValues from "../parsed_game_options_value";
+import type TestSuite from "./test_suite";
+
+const BASIC_OPTIONS_TEST_SUITE: TestSuite = {
+    tests: [
+        {
+            command: ",limit 2",
```

Comment:
These should use bot prefix

Reply:
They are replaced with bot prefix in the runner code